pydatastructs/trees/heaps.py

```diff
@@ -65,13 +67,14 @@ class BinaryHeap(Heap):
     References
     ==========
 
-    .. [1] https://en.m.wikipedia.org/wiki/Binary_heap
+    .. [1] https://en.wikipedia.org/wiki/D-ary_heap
     """
-    __slots__ = ['_comp', 'heap', 'heap_property', '_last_pos_filled']
+    __slots__ = ['_comp', 'heap', 'm', 'heap_property', '_last_pos_filled']

```

Comment:
```suggestion
    __slots__ = ['_comp', 'heap', 'd', 'heap_property', '_last_pos_filled']

```
Let's keep `d` to denote the generalisation everywhere.

Reply:
OK, it was by mistake.